index_ghost_host.html

```diff
@@ -0,0 +1,373 @@
+          <div>1 year</div>
+          <div>$9.00 USD</div>
+          <div>Transfer price no</div>
+        </div>
+      </div>
+    </section>
+    <!--Packages-->
+    <section class="packageSection">
+      <div class="packageDiv flexRow">
+        <div class="singlePackage">
+          <img class="packageImage" src="img/basic_package.png" alt="">
+          <p>Basic package</p>
+          <div class="packageProperties">
+            <ul>
+              <li>Disk space (GB)</li>
+              <li>Subdomains</li>
+              <li>Transfer (GB)</li>
+              <li>Data bases</li>
+              <li>Dasboards</li>
+              <li>Control panel & FTP</li>
+              <li>Free support</li>
+            </ul>
+            <ul class="rightColumn">
+              <li>50</li>
+              <li>50</li>
+              <li>50</li>
+              <li>1</li>
+              <li>1</li>
+            </ul>
+          </div>
+          <p>$15<span class="packagePropertiesMonth">/Mo</span></p>
+          <input class="packageButton" type="button" name="Login" value="Select plan">
+        </div>
+        <div class="singlePackage">
+          <img class="packageImage" src="img/normal_package.png" alt="">
+          <p>Normal package</p>
+          <div class="packageProperties">
+            <ul>
+              <li>Disk space (GB)</li>
+              <li>Subdomains</li>
+              <li>Transfer (GB)</li>
+              <li>Data bases</li>
+              <li>Dasboards</li>
+              <li>Control panel & FTP</li>
+              <li>Free support</li>
+            </ul>
+            <ul class="rightColumn">
+              <li>100</li>
+              <li>100</li>
+              <li>100</li>
+              <li>2</li>
+              <li>2</li>
+            </ul>
+          </div>
+          <p>$30<span class="packagePropertiesMonth">/Mo</span></p>
+          <input class="packageButton" type="button" name="Login" value="Select plan">
+        </div>
+        <div class="singlePackage">
+          <img class="packageImage" src="img/big_package.png" alt="">
+          <p>Big package</p>
+          <div class="packageProperties">
+            <ul>
+              <li>Disk space (GB)</li>
+              <li>Subdomains</li>
+              <li>Transfer (GB)</li>
+              <li>Data bases</li>
+              <li>Dasboards</li>
+              <li>Control panel & FTP</li>
+              <li>Free support</li>
+            </ul>
+            <ul class="rightColumn">
+              <li>150</li>
+              <li>150</li>
+              <li>150</li>
+              <li>4</li>
+              <li>4</li>
+            </ul>
+          </div>
+          <p>$40<span class="packagePropertiesMonth">/Mo</span></p>
+          <input class="packageButton" type="button" name="Login" value="Select plan">
+        </div>
+        <div class="singlePackage">
+          <img class="packageImage" src="img/biggest_package.png" alt="">
+          <p>Biggest package</p>
```

Comment:
it look mostly like span
```suggestion
          <span>Biggest package</span>
```